.github/workflows/build-images.yml

```diff
@@ -21,24 +21,25 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        service: ["php-apache", "php-apache-nvm"]
         php: [7.3, 7.4, "8.0"]
     steps:
       - name: Checkout code
         uses: actions/checkout@v2.3.5
 
       - name: Validate docker images
         if: ${{ github.event_name == 'pull_request' }}
-        run: docker build services/php-apache/ --build-arg PHP_VERSION=${{ matrix.php }}
+        run: docker build services/${{ matrix.service }}/ --build-arg PHP_VERSION=${{ matrix.php }}
```

Comment:
You can see in the checks for this PR that [a `php-apache-nvm` image is created for every `php` version](https://github.com/ChromaticHQ/docker-images/actions/runs/1363402755), in addition to the expected `php-apache` images.